product_docs/docs/pgd/5/routing/readonly.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Read-Only Routing with PGD Proxy
+navTitle: Read-Only Routing
+---
+
+## Background
+
+By default, PGD Proxy routes connections to the currently selected write leader in the cluster. This allows the write traffic conflicts to be rapidly and consistently resolved. Just routing everything to a single node, the write leader, is a natural fit for traditional high availability deployments where system throughput is typically limited to the throughput of what a single node can handle.
+
+But for some use cases, this behavior also means that clients that are only querying the data are also placing a load on the current write leader. It's possible this writer leader could be equally well served by one of the non-write leader nodes in the cluster. 
+
+If you could move traffic that was read-only queries to the non-write leader nodes, you could, at least in theory, handle a multiple of single node throughput. An approach like this would typically require changes to applications knowing details of cluster topology and current node status to detect write lead.
```

Comment:
I had a hard time following this sentence.

Reply:
Which one?

Reply:
Last sentence.

Reply:
```suggestion
If you could move traffic that was read-only queries to the non-write leader nodes, you could, at least in theory, handle a throughput which could be a multiple of a single nodes capability. 
An approach like this would, though, usually require changes to applications so that they were aware of details of cluster topology and the current node status to detect the write leader.
```